packages/cozy-stack-client/src/CozyStackClient.spec.js

```diff
@@ -597,6 +597,41 @@ describe('CozyStackClient', () => {
       const newToken = client.refreshToken()
       await expect(newToken).rejects.toThrow()
     })
+
+    fit(`should NOT try to refresh several times in parallele`, async () => {
```

Comment:
```suggestion
    it(`should NOT try to refresh several times in parallel`, async () => {
```

Reply:
it's @nono fault! I was late on a meeting, I forgot to remove the `fit` 🤣